### ISSUE TITLE
Use latest charter template

### ISSUE THREAD

**Issue by csarven** [solid/solid-wg-charter#2]
https://github.com/w3c/charter-drafts

**Comment by pchampin** [comment]
Yes, the charter template has evolved quite a lot since we copied it to draft this proposal :-/
Let's defer the update until there are no other pending PR on the content.
